### ISSUE TITLE
[Accessibility] Missing the "linkLabel1 link" level for LinkLabel control in the inspect tool 

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#4120]
* More info
1. The issue also can **reproduce on .NET 5.0 RC2.**
2. The issue can't reproduce on .NET SDK 3.1 and .NET 4.8.

* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20514.15

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**
Missing the 'linkLabel1' link level for LinkLabel control in the inspect tool with UI Automation mode.

**Minimal repro:**
1.  Create a .NET Core project with a LinkLabel control.
2. Build and run the core project.
3. Observe the linkLabel1 control in the Inspect tool on UI Automation mode.

Or
1. Open the attached project.
2. Build and run the core project.
3. Observe the linkLabel1 control in the Inspect tool on UI Automation mode.

**Expected behavior:**
Have the correct levels displayed for linkLabel1 control in the Inspect tool.
![ExpectResults](https://user-images.githubusercontent.com/38325459/96218330-1921a800-0fb7-11eb-8c03-faa4575ca5c5.PNG)

**Actual Behavior:**
Missing the "linkLabel1 link" level for LinkLabel control in the inspect tool.
![NewIssueActual](https://user-images.githubusercontent.com/38325459/96218338-1e7ef280-0fb7-11eb-97a2-9e34b7ac47e5.PNG)

**Comment by Tanya-Solyanik** [comment]
@Zheng-Li01 - great bug! Do you happen to know when it regressed?
@merriemcgaw  - we are not showing LINK element of the link label... so the user can't click it, we should service it....

**Comment by Zheng-Li01** [comment]
@Tanya-Solyanik, verified the issue from 5.0.rc2 to 5.0.100-preview.1.20155.7(https://dotnet.microsoft.com/download/dotnet/5.0), the issue reproduce all the time.

**Comment by Jenna-Zheng** [comment]
Verified this issue with .NET SDK 6.0.100-alpha.1.20562.2, it is fixed that have the correct levels displayed for linkLabel1 control in the Inspect tool.
![image](https://user-images.githubusercontent.com/64949616/99021828-7a2fa200-259c-11eb-9eab-7dd0eadf220f.png)

**Comment by Zheng-Li01** [comment]
Verified this issue in .Net SDK 5.0.101 build, this issue is fixed.
